src/gobjects.rs

```diff
@@ -1,5 +1,5 @@
 use std::ascii::AsciiExt;
-use std::collections::HashMap;
+use std::collections::{HashMap};
```

Comment:
There was `HashSet` there but I replaced it with a sorted vector later. I'll clean this up.